client/js/transpecos.js

```diff
@@ -54,32 +54,13 @@
 
```

Comment:
git rm client/js/transpecos.js -- this shouldn't be in source control. It's in your gitignore now but you still need to remove it